src/sql/orderbook/batch_rewards.sql

```diff
@@ -0,0 +1,42 @@
+with hashed_observations as (SELECT block_number,
+                                    tx_hash,
+                                    solver,
+                                    effective_gas_price * gas_used as execution_cost,
+                                    surplus,
+                                    fee,
+                                    auction_id
+                             FROM settlement_observations so
+                                    LEFT OUTER JOIN LATERAL (
+                               SELECT tx_hash, solver, tx_nonce, tx_from
+                               FROM settlements s
+                               WHERE s.block_number = so.block_number
+                                 AND s.log_index > so.log_index
+                               ORDER BY s.log_index
+                               LIMIT 1
+                               ) AS settlement ON TRUE
+                                    JOIN auction_transaction
+                                         ON settlement.tx_from = auction_transaction.tx_from
+                                           AND settlement.tx_nonce = auction_transaction.tx_nonce
+                             WHERE block_number > {{start_block}} AND block_number <= {{end_block}}),
+
+     reward_data as (select
+                       -- observations
+                       block_number,
+                       concat('0x', encode(tx_hash, 'hex'))  as tx_hash,
+                       concat('0x', encode(solver, 'hex'))  as solver,
+                       execution_cost,
+                       surplus,
+                       fee,
+                       (surplus + fee - reference_score) / pow(10, 18) as reward_eth,
```

Comment:
How do we convert `surplus` and `fees` into ETH here?

Reply:
I was told that everything was already in ETH and that is a fundamental assumption we make of the orderbook. Cc @fhenneke